### ISSUE TITLE
Exported record fields should be writable in Oberon07

### ISSUE THREAD

**Issue by Spivoxity** [Spivoxity/obc-3#4]
In Oberon07, with
```
MODULE M1; TYPE rec = RECORD x: INTEGER; END; VAR s*: rec; ...

MODULE M2; IMPORT M1; VAR r: M1.rec; BEGIN r.x := 7; M1.s.x := 8; ...
```
the assignment to r.x should be allowed, but that to M1.s.x should not. At present, both are forbidden.

**Comment by Spivoxity** [comment]
Note that in Wirth's implementation, an assigment M1.p^.x := 9 is also forbidden, where M1.p is an exported pointer variable, though the sequence
```
q := M1.p; q^.x := 9
```
is allowed. That looks like a bug.

**Comment by Spivoxity** [comment]
Resolved in 3.0.1
